javascript.md

```diff
@@ -0,0 +1,18 @@
+## Draft JavaScript Guidelines
+
+### General
+*  Use [Standard JS](https://standardjs.com/) to lint your code. A conistent approach to code layout, spacing and formatting to code makes it easier to switch between projects. By [adding it as a dev dependency](https://standardjs.com/index.html#install) it can be easily used in the terminal, [Webstorm](https://blog.jetbrains.com/webstorm/2017/04/using-javascript-standard-style/), and [Visual Studio](https://marketplace.visualstudio.com/items?itemName=chenxsan.vscode-standardjs).
+* Test your code for compatibility with your target browsers using a tool such as [Browserstack](https://www.browserstack.com/?utm_medium=ppc&utm_source=bing&utm_term=browserstack&device=c&network=s&matchtype=e&campaign=Search-Brand-EMEA&utm_agn=BrowserStack-Alpha).
+* Avoid polluting the global state where possible. I.e. Take steps such as using a build tool to combine related files or, at a minimum [write to window.somenamespace.x rather than window.x](https://www.zendesk.com/blog/keep-javascript-libraries-from-colliding/).
+* Think about how you will collect information about errosr on the client and consider the use of a tool such as [airbrake-js](https://github.com/airbrake/airbrake-js).
```

Comment:
Can we say something more definite than "think about"?  Is there some specific guidance we can provide on logging errors?